src/calculateRentalCost.js

```diff
@@ -28,7 +28,23 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const costPerDay = 40;
+  const threeDaysDiscount = 20;
+  const sevenDaysDiscount = 50;
+  const threeDays = 3;
+  const sevenDays = 7;
+  const totalCost = days * costPerDay;
+
+  switch (true) {
+    case days >= sevenDays:
+      return totalCost - sevenDaysDiscount;
+
+    case days >= threeDays:
+      return totalCost - threeDaysDiscount;
+
+    default:
+      return totalCost;
+  }
```

Comment:
Better use switch when you have to completely compare your value with another. In your case, it is better to use if, but without nesting as indicated in previous comments
```suggestion
    if (days >= sevenDays) {
      return totalCost - sevenDaysDiscount;
   }
   
   if (days >= threeDays) {
      return totalCost - threeDaysDiscount;
   }
   
   return totalCost;
  }
```